src/test/java/com/google/api/codegen/testdata/php_mock_service_library.baseline

```diff
@@ -0,0 +1,68 @@
+============== file: tests/unit/Example/Library/V1/MockLabelerImpl.php ==============
+<?php
+/*
+ * Copyright 2016, Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+namespace Google\Tests\Example\Library\V1;
+
+use \Google\GAX\GrpcCredentialsHelper;
+use \Google\GAX\Testing\MockStubTrait;
+use \PHPUnit_Framework_TestCase;
+use \google\tagger\v1\LabelerGrpcClient;
+
+class MockLabelerImpl extends LabelerGrpcClient
+{
+    use MockStubTrait;
+}
+============== file: tests/unit/Example/Library/V1/MockLibraryServiceImpl.php ==============
+<?php
+/*
+ * Copyright 2016, Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+namespace Google\Tests\Example\Library\V1;
+
+use \Google\GAX\GrpcCredentialsHelper;
+use \Google\GAX\Testing\MockStubTrait;
+use \PHPUnit_Framework_TestCase;
+use \google\example\library\v1\LibraryServiceGrpcClient;
```

Comment:
The MockStubTrait and LibraryServiceGrpcClient are, the others not. The unused imports are cleaned up in artman by formatting tools - is that still the preferred way to handle them? Or do we want to try to remove them from the generated output